code/modules/antagonists/fugitive/old_god.dm

```diff
@@ -0,0 +1,70 @@
+
+/mob/camera/yalp_elor
+	name = "Yalp Elor"
+	real_name = "Yalp Elor"
+	desc = "An old, dying god. It's power has been severely sapped ever since it has lost it's standing in the world."
+	icon = 'icons/mob/cameramob.dmi'
+	icon_state = "yalp_elor"
+	invisibility = INVISIBILITY_OBSERVER
+	call_life = TRUE
+	var/lastWarning = 0
+	var/given_acolytes = FALSE //so the life deletes yalp when it runs out of cultists
+	var/list/the_faithful = list() //list of the other fugitives
```

Comment:
Reminder to myself: now that I have fugitives in a team, I can move yalp's powers to affect others in the team instead of this list BS